bucket/x64dbg.json

```diff
@@ -1,52 +1,87 @@
 {
-    "version": "2021-10-31_10-53",
+    "version": "2021-11-03_11-01",
     "description": "x64/x32 debugger",
     "homepage": "https://x64dbg.com/",
     "license": "GPL-3.0-only",
-    "url": "https://github.com/x64dbg/x64dbg/releases/download/snapshot/snapshot_2021-10-31_10-53.zip",
-    "hash": "2e04d8fc7ca56648ef3e990c1993c6eb606f9db807989f61921c3d9f316610ea",
-    "pre_install": [
-        "'release\\x96dbg.ini', 'release\\x32\\x32dbg.ini', 'release\\x64\\x64dbg.ini' | ForEach-Object {",
-        "    if (!(Test-Path \"$persist_dir\\$_\")) { New-Item \"$dir\\$_\" | Out-Null }",
-        "}"
-    ],
-    "bin": [
-        "release\\x96dbg.exe",
-        "release\\x32\\x32dbg.exe",
-        "release\\x64\\x64dbg.exe"
-    ],
-    "shortcuts": [
-        [
-            "release\\x96dbg.exe",
-            "x96dbg"
-        ],
-        [
-            "release\\x32\\x32dbg.exe",
-            "x32dbg"
-        ],
-        [
-            "release\\x64\\x64dbg.exe",
-            "x64dbg"
-        ]
-    ],
-    "persist": [
-        "release\\x96dbg.ini",
-        "release\\x32\\x32dbg.ini",
-        "release\\x32\\memdumps",
-        "release\\x32\\garbage",
-        "release\\x32\\db",
-        "release\\x32\\plugins",
-        "release\\x64\\x64dbg.ini",
-        "release\\x64\\memdumps",
-        "release\\x64\\garbage",
-        "release\\x64\\db",
-        "release\\x64\\plugins"
-    ],
+    "architecture": {
+        "64bit": {
+            "url": "https://downloads.sourceforge.net/project/x64dbg/snapshots/snapshot_2021-11-03_11-01.zip",
+            "hash": "d3c9c3161eda5a95a71ac77dce3b860231e95c8c57b9088751430a86aecd0e8f",
+            "pre_install": [
+                "'release\\x96dbg.ini', 'release\\x32\\x32dbg.ini', 'release\\x64\\x64dbg.ini' | ForEach-Object {",
+                "    if (!(Test-Path \"$persist_dir\\$_\")) { New-Item \"$dir\\$_\" | Out-Null }",
+                "}"
+            ],
```

Comment:
Since these are same for both architecture, move them outside, only keep `bin` and `shortcuts` under arch.

New-Item create directory by default, so add an `-ItemType File` param.